applications/CompressiblePotentialFlowApplication/tests/naca0012_small_adjoint_test/naca0012_test_shape_sensitivity_probe1_ref.dat

```diff
@@ -1,3 +1,3 @@
 # Results for "element" with Id # 6 at position: x: 0.49961; y: 0.09728; z: 0
 # time SHAPE_SENSITIVITY_X SHAPE_SENSITIVITY_Y SHAPE_SENSITIVITY_Z
-0.0 -0.1811648561717486 -2.2783700504916142 0.0
+0.0 -0.18115769295952455 -2.2783681753440868 0.0
```

Comment:
Same comment.

Reply:
This is solved in https://github.com/KratosMultiphysics/Kratos/pull/4668